backend/src/main.py

```diff
@@ -1,28 +1,21 @@
 import logging
+import os
 import sys
 from contextlib import asynccontextmanager
 
-from dotenv import load_dotenv
-from fastapi import Depends, FastAPI, HTTPException
-from sqlalchemy.orm import Session
-
-from src.crud import (
-    create_listing,
-    create_user,
-    delete_listing,
-    delete_user,
-    read_listing,
-    read_listings,
-    read_user,
-    read_user_by_email,
-    read_users,
-    update_listing,
-    update_user,
-)
-from src.db import get_db
-from src.schemas import ListingCreate, ListingRead, UserCreate, UserRead
-
-load_dotenv()
+import django
+from asgiref.sync import sync_to_async
+from django.core.wsgi import get_wsgi_application
+from fastapi import FastAPI, HTTPException
+from fastapi.middleware.wsgi import WSGIMiddleware
+
+# Initialize Django app before importing models
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "src.fairbnb.settings")
+django.setup()
+
+from src.crud import PropertyDBClient, UserDBClient  # noqa: E402
+from src.schemas import PropertyCreate, PropertyRead, UserCreate, UserRead  # noqa: E402
+from src.seed import seed_database  # noqa: E402
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
```

Comment:
This should be based on DEBUG from settings as well, like info if not debug and debug if debug